tests/Doctrine/Tests/DBAL/Driver/AbstractSQLServerDriverTest.php

```diff
@@ -54,4 +56,40 @@ protected function getDatabasePlatformsForVersions()
             array('12', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
         );
     }
+
+    protected function getDriver() : AbstractSQLServerDriver
+    {
+        return static::createDriver();
+    }
+
+    protected function checkForSkippingTest(AbstractSQLServerDriver $driver) : void
+    {
+        $this->markTestSkipped('The test is only for the sqlsrv and the pdo_sqlsrv drivers');
+    }
+
+    /**
+     * @param mixed[] $driverOptions
+     */
+    protected function getConnection(AbstractSQLServerDriver $driver, array $driverOptions) : DriverConnection
+    {
+        return $driver->connect(
+            [
+                'host' => $GLOBALS['db_host'],
+                'port' => $GLOBALS['db_port'],
+            ],
+            $GLOBALS['db_username'],
+            $GLOBALS['db_password'],
+            $driverOptions
+        );
+    }
+
+    public function testConnectionOptions() : void
+    {
+        $driver = $this->getDriver();
+        $this->checkForSkippingTest($driver);
+        $connection = $this->getConnection($driver, ['APP' => 'APP_NAME']);
+        $result     = $connection->query('select APP_NAME() as app')->fetch();
```

Comment:
You can use `fetchColumn()` instead of `fetch()` and remove the "app" alias for simplicity.

Reply:
Please use upper case for SQL keywords for better readability (`'SELECT'`).